developing.adoc

```diff
@@ -46,40 +46,40 @@ For more information, see the following:
 . link:https://github.com/golang/go/wiki/Modules[Introduction to Go modules]
 . link:https://blog.golang.org/using-go-modules[Using Go modules]
 
-[[running-integration-tests]]
-== Running integration tests
+[[running-e2e-tests]]
+== Running e2e tests
 
-We have automated integration tests which keep CodeReady Containers in shape.
+We have automated e2e tests which keep CodeReady Containers in shape.
```

Comment:
In the documentation, I'm not so sure about using 'e2e' as is. Maybe explicitly spell 'end-to-end (e2e)', at least the first few times it appears?

Reply:
I don't want to replace it completely with 'end-to-end', as in code we use `e2e`. But I did the "definition" as you said: spelled it out the first time with bracketed "from now on" form. Looks more natural now!